registry.json

```diff
@@ -2115,5 +2115,20 @@
     "telegram":"t0m3kf",
     "operatorName":"",
     "github":"t0m3kf"
+  },
+  {
+    "ticker":"PANL",
+    "poolId":"05e11c24a4de15366c6d548f8847589164d02260b6a059a08c22be2a",
+    "social":{
+      "facebook":"",
+      "twitter":"https://twitter.com/PANL_stakepool",
```

Comment:
Hey. how about we get some action on twitter. just to make sure you are a real person.

Reply:
Can do Leo, just made our first post and updated some info.  Putting together something about the Africa news now.

Thanks for your hard work!